src/components/Pagination/index.js

```diff
@@ -28,81 +29,47 @@ class Pagination extends React.Component {
-      <li onClick={ this.setPage.bind( this, pager.currentPage - 1 ) }>
-        <a href="#" aria-label="Previous">
-          <span aria-hidden="true">{ this.props.labels.previous }</span>
-        </a>
-      </li>;
+      <PaginationLink href="#" label={ this.props.labels.previous } ariaLabel="Previous"
+                      helper={ this.props.helper } /> :
+      <PaginationLink href="#" label={ this.props.labels.previous } ariaLabel="Previous"
+                      helper={ this.props.helper } page={pager.currentPage - 1}/>;
   }
   nextPageLink( pager ) {
     return pager.isLastPage() ?
-      <li className="disabled">
-        <a href="#" aria-label="Next">
-          <span aria-hidden="true">{ this.props.labels.next }</span>
-        </a>
-      </li> :
-      <li onClick={ this.setPage.bind( this, pager.currentPage + 1 ) }>
-        <a href="#" aria-label="Next">
-          <span aria-hidden="true">{ this.props.labels.next }</span>
-        </a>
-      </li>;
+      <PaginationLink href="#" label={ this.props.labels.next } ariaLabel="Next"
+                      helper={ this.props.helper } /> :
+      <PaginationLink href="#" label={ this.props.labels.next } ariaLabel="Next"
+                      helper={ this.props.helper } page={ pager.currentPage + 1 } />
   }
   firstPageLink( pager ) {
     return pager.isFirstPage() ?
-      <li className="disabled">
-        <a href="#" aria-label="Previous">
-          <span aria-hidden="true">{ this.props.labels.first }</span>
-        </a>
-      </li> :
-      <li onClick={ this.setPage.bind( this, 0 ) }>
-        <a href="#" aria-label="Previous">
-          <span aria-hidden="true">{ this.props.labels.first }</span>
-        </a>
-      </li>;
+        <PaginationLink href="#" label={ this.props.labels.first } ariaLabel="First"
+                        helper={ this.props.helper } /> :
+        <PaginationLink href="#" label={ this.props.labels.first } ariaLabel="First"
+                        helper={ this.props.helper } page={ 0 } />;
   }
   lastPageLink( pager ) {
     return pager.isLastPage() ?
-      <li className="disabled">
-        <a href="#" aria-label="Next">
-          <span aria-hidden="true">{ this.props.labels.last }</span>
-        </a>
-      </li> :
-      <li onClick={ this.setPage.bind( this, pager.total - 1 ) }>
-        <a href="#" aria-label="Next">
-          <span aria-hidden="true">{ this.props.labels.last }</span>
-        </a>
-      </li>;
+        <PaginationLink href="#" label={ this.props.labels.last } ariaLabel="Last" /> :
+        <PaginationLink href="#" label={ this.props.labels.last } ariaLabel="Last"
+                        helper={ this.props.helper } page={ pager.total - 1 } />;
   }
   pages( pager ) {
     var pages = pager.pages();
     var elements = [];
 
     forEach( pages, function( pageNumber ) {
-      elements.push( this.page( pager, pageNumber ) );
+      var className = pageNumber === pager.currentPage ? "active" : "";
```

Comment:
I'm not used to that because if the statement spreads over more than one line then it doesn't look really nice.